### PR TITLE
Inspect $FISH_VERSION variable instead of $version

### PR DESCRIPTION
**Pull request by lfiolhais**
Per fish-shell/fish-shell@95635a5

### CODE REVIEW

**Comment by derekstavis** [comment]
Hey, thanks for the PR! :+1:

**Comment by lfiolhais** [comment]
No problem, we would have to do this when the next version shipped so we might as well do it now.

**Comment by bobthecow** [comment]
:+1:

**Comment by coderstephen** [comment]
Should the installer check both `$FISH_VERSION` and `$version`? I don't recall `$FISH_VERSION` being defined for Fish < 2.3.0.

**Comment by lfiolhais** [comment]
Didn't think of that @coderstephen. I'll add it now.

File: bin/install [hunk 1]
```diff
@@ -54,8 +54,8 @@ function report -a what message
 end
 
 function fish_version_compatible
```
Comment (bobthecow) [inline bin/install]: set -q FISH_VERSION; or set -l FISH_VERSION $version

**Comment by bobthecow** [comment]
:+1: mind rebasing so we can get a clean merge?

**Comment by scorphus** [comment]
:+1: let us go for it

**Pull request merged**